openedx/core/djangoapps/site_configuration/templatetags/configuration.py

```diff
@@ -35,12 +35,12 @@ def platform_name():
 
 
 @register.simple_tag(name="favicon_path")
-def favicon_path(default=getattr(settings, 'FAVICON_PATH', 'images/favicon.ico')):
+def favicon_path():
```

Comment:
Is this favicon_path function actually used anywhere?

Hasn't it been replaced with this, shown above in main.html?

```
<% favicon_url = branding_api.get_favicon_url() %>
<link rel="icon" type="image/x-icon" href="${favicon_url}"/>
```

Reply:
Good call. 

actually, there is also a Django base template in LMS which is used by the course wiki app. Wiki pages inherit `main_django.html` instead of `main.html`. 

In the [main_django](https://github.com/edx/edx-platform/blob/master/lms/templates/main_django.html#L8) base template, this `favicon_path` is used a Django template tag.

Reply:
@saadyousafarbi  ^